scripts/coretemp/download-language-packs.ps1

```diff
@@ -0,0 +1,42 @@
+$language_pack_urls = @(
```

Comment:
Slight chance of optimization here: making an array of just the `zh-CN` part, the rest of the URL is identical for all entries.